src/main/java/eu/bbmri_eric/negotiator/plugins/resourcesync/listeners/JobEventManager.java

```diff
@@ -0,0 +1,42 @@
+package eu.bbmri_eric.negotiator.plugins.resourcesync.listeners;
+
+import eu.bbmri_eric.negotiator.database.model.DiscoveryServiceSynchronizationJob;
+import eu.bbmri_eric.negotiator.database.model.DiscoveryServiceSyncronizationJobStatus;
+import eu.bbmri_eric.negotiator.database.repository.DiscoveryServiceSynchronizationJobRepository;
+import eu.bbmri_eric.negotiator.events.DiscoveryServiceSynchronizationEvent;
+import eu.bbmri_eric.negotiator.exceptions.EntityNotStorableException;
+import eu.bbmri_eric.negotiator.service.DiscoveryServiceClient;
+import lombok.extern.apachecommons.CommonsLog;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationListener;
+import org.springframework.stereotype.Component;
+
+@CommonsLog
+@Component
+public class JobEventManager implements ApplicationListener<DiscoveryServiceSynchronizationEvent> {
+  @Autowired
+  private DiscoveryServiceSynchronizationJobRepository discoveryServiceSynchronizationJobRepository;
+
+  @Autowired private DiscoveryServiceClient bbmriDirectoryServiceSyncClient;
```

Comment:
please do not autowire specific implementations

Reply:
fixed in fb760484e9b2dbec0c0c5b584decfbd60305ad4d